dispatchr-ruby/spec/controllers/request_items_controller_spec.rb

```diff
@@ -0,0 +1,116 @@
+require 'rails_helper'
+
```

Comment:
Yeah I just realized I don't either. I tried to create a test but it was just returning errors, unable to hit the endpoint for some reason. I would just ensure that the # of items returned == # of entries in that table.